### DISCUSSION TITLE
How to get daily price change data of your portfolio using Portfolio

### DISCUSSION THREAD

**Discussion by Praj-17** [mbk-dev/okama#72] [Q&A]
Hello I want to get data of my portfolio in the following format

Open Close Low High Volume

Inorder to be able to plot the candlestick pattern of my portfolio. How can I do that?

**Comment by chilango74** [comment]
> Open Close Low High Volume

Okama project is created to anylize long term return time series in terms of Modern Portfolio Theory (MPT). It uses monthly close data (mostly).  In the database we have daily data also. However, we don't go deeper. There is no open, high, low data...

Unfortunately, with our data you will not be able to plot candlesticks.
